### PR TITLE
- read file from resource

### PR DESCRIPTION
**Pull request by AndrewQA88**
- add custom error message

### CODE REVIEW

File: src/test/java/exercises/filereader/ReadFileFromResources.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        ReadFileFromResources readFileFromResources = new ReadFileFromResources();
+        String fileName = "test.txt";
+        String brokenName = "ert.com";
+        InputStream is = readFileFromResources.getFileFromResourceAsStream(fileName);
+        printInputStream(is);
+    }
+
+    public InputStream getFileFromResourceAsStream(String fileName) {
+        ClassLoader classLoader = getClass().getClassLoader();
```
Comment (iikarak) [inline src/test/java/exercises/filereader/ReadFileFromResources.java:21]: update to have file inside folder in resources and don`t use full path, 
and read about File.separator why this is used?

File: src/test/java/exercises/filereader/ReadFileFromResources.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        InputStream inputStream = classLoader.getResourceAsStream(fileName);
+        if (inputStream == null) {
+            throw new IllegalArgumentException("file is not found :-( //" + fileName);
+        } else {
+            return inputStream;
+        }
+    }
+
+    public static void printInputStream(InputStream is) {
+        try (InputStreamReader streamReader = new InputStreamReader(is, StandardCharsets.UTF_8);
```
Comment (iikarak) [inline src/test/java/exercises/filereader/ReadFileFromResources.java:31]: rewrite to not use try with resources

**Review by iikarak — CHANGES_REQUESTED** [review]

File: src/test/java/exercises/filereader/ReadFileFromResources.java (hunk 1 above)
Comment (AndrewQA88) [reply src/test/java/exercises/filereader/ReadFileFromResources.java:21]: Updated:
- added folder
- used FileSeparator to create multiplatfor's paths

File: src/test/java/exercises/filereader/ReadFileFromResources.java (hunk 2 above)
Comment (AndrewQA88) [reply src/test/java/exercises/filereader/ReadFileFromResources.java:31]: added method with try - catch - finally

**Review by iikarak — APPROVED** [review]

**Pull request merged**